### DISCUSSION TITLE
What storage location is used for the "Storage" display?

### DISCUSSION THREAD

**Discussion by EinToni** [immich-app/immich#2556] [Q&A] [answered]
Hi,

I currently have immich installed via Docker and the upload directory is split up as shown [here](https://github.com/immich-app/immich/discussions/2328#discussioncomment-5712579).
So my system with docker lives on a small 128 GB SSD.
`Thumbs`, `Upload` and `encoded-video` are on a separate 256B SSD and the actual `library` on a large HDD.

In the overview on the Immich landing page, the storage usage is now shown for my system drive:
![image](https://github.com/immich-app/immich/assets/51962051/ae5c4a24-57eb-464c-b7ed-a3de03aaedab)

What is used as the storage size reference, and may that be changed to show the size of the directory that matters the most (the library :) )?

Cheers.

**Comment by jrasm91** [comment, marked as answer]
Current statistics come from the drive that is linked to the `UPLOAD_LOCATION` folder. #2560 changes this to `UPLOAD_LOCATION/library`.

**Reply by ArtekMisiek** [reply]
@jrasm91 What if i am using mount point (Proxmox -> LXC)? 
I store LXC on system drive (with 50GB avaliable), then i mount 4TB drive to it and use that as `UPLOAD_LOCATION`. 
Immich gives me info that there is X/50GB instead of 4TB. Is there a way around this?

**Reply by bo0tzz** [reply]
That sounds like something is wrong with your mount.

**Reply by ArtekMisiek** [reply]
you are correct. For future reference:
I had wrong path in `UPLOAD_LOCATION` `./data` - instead of `../../data` (mount location). Also creating directories (`library`, `uploads`) and `chown 1005:1005` was necessary to work.
Thanks

**Comment by jlar0che** [comment]
I personally don't mind that the Storage Space shown when logged into the homepage references the system drive. But, when the API references "Storage Space" it is returning the amount of space used int the UPLOAD_LOCATION and that makes no sense. It should be -- as the OP stated -- the size of the Library of the current user.

![Untitled-1](https://github.com/user-attachments/assets/62f42083-9497-4b07-9c9f-4070c22c6fcd)

**Comment by daviddwlee84** [comment]
I'm just running a brand new app on 512GB Mac Mini using docker compose followed the [quick start page](https://immich.app/docs/overview/quick-start)
But the "Storage space 52.8 TiB of 115.1 TiB used" makes no sense 😂
![image](https://github.com/user-attachments/assets/d96aaa61-9a45-4bfe-9b5e-9cbe93b1a414)

**Reply by daviddwlee84** [reply]
- https://github.com/immich-app/immich/issues/7482
- https://github.com/immich-app/immich/issues/4318
- https://github.com/immich-app/immich/issues/8202
- https://www.reddit.com/r/immich/comments/1l0ubnx/immich_disk_storage_space_used_not_matching_the/
